standalone/standalone.tpl

```diff
@@ -2016,6 +2028,7 @@ def sdg(
                 "api_key": sdg_serving_model_api_key,
                 "endpoint": sdg_serving_model_endpoint,
                 "model": sdg_serving_model_name,
+                "pipeline_type": sdg_pipeline_type,
```

Comment:
You also need to account for the 

`elif sdg_serving_model_secret:` 

We must validate that the key `pipeline_type` is present and that the value is either simple or agentic

Reply:
addressed